docs/components/configMaps.md

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.configMaps`
+
+This grade provides the configuration maps for the gamepad in the form of component's model data.
+
+## Using this grade
+
+This can be used by passing it to the component's `gradeNames` option.
+
+``` javascript
+fluid.defaults("my.configMaps.grade", {
+    gradeNames: ["gamepad.configMaps"]
+};
+
+var configMapsInstance = my.configMaps.grade();
+```
+
+## Component Options
+
+This grade doesn't use or provide any configuration options. It only uses the model data to provide gamepad configuration
+maps to other components. However, the grade can be extended to provide custom configuration options.
+
+## Model Structure
+
+The grade stores the model data in the following format:
+
+``` javascript
```

Comment:
This should have failed the linting checks.  We typically use `snippet` as the code type for incomplete pieces of Javascript or JSON.

Reply:
Thanks, I've updated the `Gruntfile` accordingly.